gadgets/top_file/README.mdx

```diff
@@ -0,0 +1,176 @@
+---
+title: top_file
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# top_file
+
+The top_file gadget reports periodically the read/write activity by file.
+
+## Getting started
+
+Running the gadget:
+
+<Tabs groupId="env">
+    <TabItem value="kubectl-gadget" label="kubectl gadget">
+        ```bash
+        $ kubectl gadget run ghcr.io/inspektor-gadget/gadget/top_file:latest [flags]
```

Comment:
I followed what was done in https://github.com/inspektor-gadget/inspektor-gadget/pull/3263, but I don't like we are adding the `$` at the beginning and `[flags]` at the end. It prevents user from using the `Copy` functionally straightforward:

![image](https://github.com/user-attachments/assets/59c80694-cd0f-4179-ad3a-32e36b311b87)

Reply:
I think we need to clarify the use of `$` all around the documentation.